www/include/views/graphs/exportData/ExportCSVServiceData.php

```diff
@@ -98,17 +126,27 @@ function get_error($str)
 $datas = array();
 $listEmptyMetric = array();
 
-$query = "SELECT DISTINCT metric_id, metric_name FROM metrics, index_data " .
-    "WHERE metrics.index_id = index_data.id AND id = '$index' ORDER BY metric_name";
-$DBRESULT = $pearDBO->query($query);
-while ($index_data = $DBRESULT->fetchRow()) {
+$stmt = $pearDBO->prepare(
+    'SELECT DISTINCT metric_id, metric_name ' .
+    'FROM metrics, index_data ' .
+    'WHERE metrics.index_id = index_data.id AND id = :index ORDER BY metric_name'
```

Comment:
what will happen is $index is false or null ?

Reply:
I guess I can check if it is set before doing the request